homeassistant/components/upb/const.py

```diff
@@ -4,13 +4,17 @@
 
 import homeassistant.helpers.config_validation as cv
 
-CONF_NETWORK = "network"
 DOMAIN = "upb"
 
+ATTR_ADDRESS = "address"
 ATTR_BLINK_RATE = "blink_rate"
 ATTR_BRIGHTNESS = "brightness"
 ATTR_BRIGHTNESS_PCT = "brightness_pct"
+ATTR_COMMAND = "command"
 ATTR_RATE = "rate"
+CONF_NETWORK = "network"
+EVENT_UPB_LINK_CHANGED = "upb.link_changed"
```

Comment:
I'm used to seeing `{INTEGRATION}_link_changed` 
https://developers.home-assistant.io/docs/dev_101_events/#firing-events

I did a run though the codebase and found both are used so I think this is ok